java/system-test/system-test-configuration/src/test/java/sleeper/systemtest/configuration/SystemTestStandalonePropertiesTest.java

```diff
@@ -0,0 +1,67 @@
+package sleeper.systemtest.configuration;
```

Comment:
I think this will fail Checkstyle. Please check your IDE configuration.

Reply:
addressed